docker-compose.override.yaml

```diff
@@ -47,7 +47,7 @@ services:
       - "10254:10254"
       - "10255:10255"
     environment:
-      AZURE_COSMOS_EMULATOR_PARTITION_COUNT: 5
+      AZURE_COSMOS_EMULATOR_PARTITION_COUNT: 10 
```

Comment:
Why do we need more partitions?

Reply:
My understanding is we need one partition per database. Ten databases is the maximum. Currently we have five databases and i wanted to ensure the emulator would still work if we went up to ten. So it was to future proof it. Doesn't appear to affect the speed but could be dropped down to five until we add further databases.

Reply:
Updated to 5 for now